step-capstone/src/components/Event.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../styles/TimeLine.css';
+/*
+Takes data in this form:
+       {
+         finalized: true,
+         type: "event",
+         title: "Visiting Sherlock",
+         location: "221b Baker Street, London",
+         startDate: "8:30",
+         endDate: "10:30",
+         description: "Additional notes"
+       }
+*/
+export default function Flight( props ) {
```

Comment:
typo: Flight should be Event

Reply:
woops, you are right. Thank you for noticing this!